build/scripts/validate_plugin_version_bump.py

```
@@ -0,0 +1,458 @@
+        return None
+
+
+def _base_version(base_ref: str, manifest: str, repo_root: Path) -> str | None:
+    """Read the manifest version at ``base_ref`` via ``git show``.
+
+    Returns ``None`` when the manifest did not exist at the base (a new
+    plugin) or when git cannot produce it. Both cases mean "no prior version
+    to compare", which ``evaluate`` treats as a pass.
+    """
+    try:
+        proc = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)